.lighthouserc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "ci": {
+    "collect": {
+      "startServerCommand": "npm run serve",
+      "startServerReadyPattern": "Server running at http://localhost:3000/",
+      "url": [
+        "http://localhost:3000/en",
+        "http://localhost:3000/en/about",
+        "http://localhost:3000/en/download",
+        "http://localhost:3000/en/docs",
+        "http://localhost:3000/en/get-involved",
+        "http://localhost:3000/en/blog"
+      ],
+      "numberOfRuns": 1,
+      "settings": {
+        "preset": "desktop"
+      }
+    },
+    "upload": {
+      "target": "temporary-public-storage"
```

Comment:
What is this upload target?

Reply:
This helps us to store the generated reports we can access them after the Audit through the URLs in the report at the end.